core/citrus-spring/src/main/java/com/consol/citrus/common/SpringXmlTestLoader.java

```diff
@@ -40,28 +41,18 @@
  * @author Christoph Deppisch
  * @since 2.1
  */
-public class XmlTestLoader implements TestLoader, TestSourceAware {
+public class SpringXmlTestLoader implements TestLoader, TestSourceAware {
```

Comment:
this is kinda breaking, right? I think we should keep `public class XmlTestLoader extends SpringXmlTestLoader` and deprecate it.

Reply:
ok with this one.